README.md

```diff
@@ -47,146 +35,37 @@ This package implements the slave side of the simple key-value protocol used  fo
-
-Example of ```module.json``` file with ```greentea-client``` as a test dependency:
-```json
-{
-  "testDependencies": {
-    "greentea-client": "^1.0.0",
-    "utest" : "^1.10.0",
-    "unity" : "^2.1.0"
-  }
-}
-```
+* [Greentea](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-greentea)
+* [htrun](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests)
 
 ## Terms
 
 ### Test suite
-A test suite is a binary containing test cases we execute on hardware. The test suite has a beginning and an end (like your ```main()``` function would. The test suite may pass, fail or be in an error state (for example if test suite times out or there was a serial port connection problem).
-
-### Test case
-
-Preferably you will use ```utest``` to define test cases . test case has the beginning and the end. During test case execution you will use ``ùnity``` assert macros, schedule MINAR callbacks, check for timeouts in your code. Your test cases may pass, fail or be in an error state which means something went wrong and we were unable to determine exactly what that was (you may have to check the logs).
+A test suite is a binary containing test cases we execute on hardware. The test suite has a beginning and an end (like your ```main()``` function would. The test suite may pass, fail or be in an error state (for example if the test suite times out or there was a serial port connection problem).
 
 ### key-value protocol
 
-The key-value protocol (also called ```KiVi```) is a simple protocol introduced to the ```Greentea``` test tools. It is used to send simple text messages (events) between the DUT and the host. Each message consists of a _key_ and corresponding _value_ pair.
-A ```KiVi``` message is defined as a string encapsulated between double curly braces. The key and value are separated by a semicolon (```;```).
-For example: the ```{{timeout;120}}}``` string is a simple key-value message where the key "_timeout_" is associated with the value "_120_". Both ```greentea-client``` and ```Greentea``` understand this format and can detect key-value messages in a data stream. Both _key_ and _value_ are ASCII strings.
+The key-value protocol (also called ```KiVi```) is a simple protocol introduced to the ```Greentea``` test tools. It is used to send simple text messages (events) between the DUT and the host. Each message consists of a _key_ and corresponding _value_ pair. A ```KiVi``` message is defined as a string encapsulated between double curly braces. The key and value are separated by a semicolon (```;```).
+
+For example, the ```{{timeout;120}}}``` string is a simple key-value message where the key "_timeout_" is associated with the value "_120_". Both ```greentea-client``` and ```Greentea``` understand this format and can detect key-value messages in a data stream. Both _key_ and _value_ are ASCII strings.
```

Comment:
This is an example of what a KiVi message is, so the sentence "A KiVi message is defined as" should still stick with this paragraph.